index.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] It will be better to fix the linter errors instead of disabling them. This is important because linters help you write clean code.